File: spring/auto-testing-example/src/main/java/com/example/autotestingexample/presentation/MyController.java
package com.example.autotestingexample.presentation;

import com.example.autotestingexample.domain.MyService;
import com.example.autotestingexample.repository.MyMapper;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
@RequiredArgsConstructor
public class MyController {

    private final MyService myService;
    private final MyMapper myMapper;

    @GetMapping("/")
    public String hello(final Model model) {

        model.addAttribute("now", myService.getNow());
        model.addAttribute("boot", myMapper.select());
        return "index";
    }
}

File: spring/spring-boot-auto-restart/gradle-kotlin-with-devtools/src/main/kotlin/com/example/gradle_kotlin_with_devtools/MyController.kt
package com.example.gradle_kotlin_with_devtools

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.RequestMapping

@Controller
class MyController {

    @RequestMapping
    fun index(model: Model): String {
        model.addAttribute("name", "Gradle v9")
        return "index"
    }
}
